javascript/javascript1/week2/candyHelper.js

```diff
@@ -0,0 +1,53 @@
+const boughtCandyPrices = [];
+
+function addCandy(candyType, weight) {
+    let candyPrice;
+    switch (candyType) {
+        case 'sweet':
+            candyPrice = weight * 0.5;
+            break;
+        case 'chocolate':
+            candyPrice = weight * 0.7;
+            break;
+        case 'toffee':
+            candyPrice = weight * 1.1;
+            break;
+        case 'chewing-gum':
+            candyPrice = weight * 0.03;
+            break;
+        default:
+            break;
+    }
+    boughtCandyPrices.push(candyPrice);
```

Comment:
smart use of switch statement  !
primarily, i used if/else statement with includes() and indexOf() method  here but that did not work and then i also used to switch statement as you did here !